docs/.gitignore

```
@@ -0,0 +1,2 @@
+#Sphinx documentation
+!requirements.txt
```

No need for the comment. It doesn't add any value here.
```suggestion
!requirements.in
!requirements.txt
```

---

okay, I fix